src/test/resources/features/ContactUs.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Contact Us
+
+  Scenario: Find, fill and submit successfully the Contact Us form.
+
+    Given The user is on the website
+    When The user clicks on the Contact Us button
+    And The user enters data to the form
```

Comment:
The `The user fills the registration form` step could be generalized and be used here